### PR TITLE
Feature/cpr cloudsat updated

### PR DESCRIPTION
**Pull request by imoradi**
## Description

This PR makes some small changes in the current CloudSat CPR converter to include Layer and height.

## Issue(s) addressed

Resolves #<issue_number>

## Impact

Expected impact on downstream repositories:

## Checklist

- [ x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x ] I have run the unit tests before creating the PR

### CODE REVIEW

**Comment by BenjaminRuston** [comment]
@imoradi I have changed the base branch to my PR this will allow easier view of the changes

File: src/hdf5/read_cloudsat.py [hunk 1]
```diff
@@ -77,6 +77,7 @@ def read_cloudsat_hdf_file(fname):
     xid = vs.find('start_time')
     start_time = np.array(vs.attach(xid)[:]).flatten()
     start_time = datetime.datetime.strptime(start_time[0], '%Y%m%d%H%M%S')
+    #start_time = datetime.datetime(2022, 2, 16, start_time.hour, start_time.minute, start_time.second)
```
Comment (BenjaminRuston) [inline src/hdf5/read_cloudsat.py]:
```suggestion
```

**Review by BenjaminRuston — APPROVED** [review]
thanks @imoradi

**Review by fcvdb — APPROVED** [review]
Thanks Isaac!

**Pull request merged**